.github/workflows/test-application.yaml

```diff
@@ -86,7 +90,7 @@ jobs:
                       phpcr-transport: jackrabbit
                       dependency-versions: 'lowest'
                       php-extensions: 'ctype, iconv, mysql, gd'
-                      tools: 'composer:v1'
+                      tools: 'composer:v2'
```

Comment:
required to install new packages: https://blog.packagist.com/deprecating-composer-1-support/

Reply:
To make it clear this would mean newer versions can only be installed when using composer v2. What do you think?

Reply:
As discussed with @chirimoya we will fix the 8.1 issues on our side in the 2.2 and 2.3 branches. But only the 2.4 branch we will use the fork in 2.2 and 2.3. So with begin of Sulu 2.4 we require composer 2.